lib/plug/quiet_logger.ex

```diff
@@ -8,9 +8,18 @@ defmodule Plug.QuietLogger do
     %{log: log, path: path}
   end
 
+  def call(%{request_path: path} = conn, %{log: :info, path: paths}) when is_list(paths) do
+    cond do
+      path in paths -> conn
+      true -> Plug.Logger.call(conn, :info)
+    end
+
+    conn
```

Comment:
You don't need this at the end of the function, you can safely rely on what's returned from the previous `cond/if` statement 🙂